controller/src/main/java/io/pravega/controller/store/stream/records/EpochRecord.java

```diff
@@ -37,25 +38,52 @@
 
     private final int epoch;
     /**
-     * Reference epoch is either the same as epoch or the epoch that originated a chain of duplicates 
-     * that includes this epoch. If we look at it as a graph, then it is a tree of depth one, where 
+     * Reference epoch is either the same as epoch or the epoch that originated a chain of duplicates
+     * that includes this epoch. If we look at it as a graph, then it is a tree of depth one, where
      * the root is the original epoch and the children are duplicates.
      */
     private final int referenceEpoch;
     private final ImmutableList<StreamSegmentRecord> segments;
     private final long creationTime;
     @Getter(AccessLevel.PRIVATE)
     private final Map<Long, StreamSegmentRecord> segmentMap;
-    
+    /**
+     * Number of splits performed as part of scale operation.
+     */
+    private final long splits;
```

Comment:
is it number of splits in this scale operation or overall so far?

Reply:
It is overall so far, since the Epoch Record will maintain only cumulative counts.

Reply:
please make that explicit in the comment above that its cumulative count

Reply:
Sure.